direct/data/datasets_config.py

```diff
@@ -37,9 +40,17 @@ class SensitivityMapEstimationTransformConfig(BaseConfig):
     sensitivity_maps_gaussian: Optional[float] = 0.7
 
 
+@dataclass
+class AugmentationTransformConfig(BaseConfig):
+    rescale: Optional[tuple[int, ...]] = None
+    rescale_mode: Optional[RescaleMode] = RescaleMode.NEAREST
+    rescale_2d_if_3d: Optional[bool] = False
+    pad: Optional[tuple[int, ...]] = None
+
+
 @dataclass
 class RandomAugmentationTransformsConfig(BaseConfig):
-    random_rotation_degrees: Tuple[int, ...] = (-90, 90)
+    random_rotation_degrees: tuple[int, ...] = (-90, 90)
     random_rotation_probability: float = 0.0
```

Comment:
Is this not tuple[int, int]?

Reply:
No it can be of any length, eg. (90, ) or (-90, -180, 90), etc